templates/query.html

```diff
@@ -0,0 +1,98 @@
+            <select class="form-control" id="model-type-input" name="model-type-input">
+                <option value="analogy">analogy</option>
+                <option value="complex">complex</option>
+                <option value="distmult">distmult</option>
+                <option value="hole">hole</option>
+                <option value="rescal">rescal</option>
+                <option value="transd">transd</option>
+                <option value="transe">transe</option>
+                <option value="transh">transh</option>
+                <option value="transr">transr</option>
+            </select>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-md-6 mb-3">
+            <label for="out-input">Path to models output directory</label>
+            <input type="text" class="form-control" id="out-input" name="out-input" value="sys/data/processed/">
+        </div>
+        <div class="col-md-6 mb-3">
+            <label for="glove-dir-input">Path to GloVe embeddings model file</label>
+            <input type="text" class="form-control" id="glove-dir-input" name="glove-dir-input" placeholder="Not required" disabled>
+        </div>
+    </div>
+    <div class="mb-3">
+        <div class="form-group">
+            <label for="run-input">Identify choice of action, fitting a model or predicting from it</label>
+            <select class="form-control" id="run-input" name="run-input">
+                <option value="analogy">fit</option>
+                <option value="complex">model</option>
+            </select>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-md-4 mb-3">
+            <label for="threshold-input">Threshold</label>
+            <input type="number" class="form-control" id="threshold-input" name="threshold-input" value="10">
+        </div>
+        <div class="col-md-4 mb-3">
+        	<label>Verbosity</label><br>
+            <div class="form-check form-check-inline">
+                <input class="form-check-input" type="radio" name="verbosity-input" id="verbose-yes" value="True">
+                <label class="form-check-label" for="verbose-yes">True</label>
+            </div>
+            <div class="form-check form-check-inline">
+                <input class="form-check-input" type="radio" name="verbosity-input" id="verbose-no" value="False" checked>
+                <label class="form-check-label" for="verbose-no">False</label>
+            </div>
+        </div>
+        <div class="col-md-4 mb-3">
+            <label for="num-top-input">Number ot top matches</label>
+            <input type="number" class="form-control" id="num-top-input" name="num-top-input" value="0.1">
+        </div>
+    </div>
+    <div class="mb-3">
+        <label for="search-input">Searches to execute. Either a path to a CSV containing triples or a triple in the format:<br> <code>"subject terms ::: relation terms ::: object terms"</code></label>
```

Comment:
Nothing wrong here, but @gallupBenRyan is the prediction piece? if so, the `network-features` piece i developed is going to need more than what I output, since I only output one entity. Do I need to get your a triple?

Reply:
The idea here was to use that seed to start a `map_queries.py` request the old-fashioned way - giving it an entity and letting it predict triples around that entity. Those would then get sent to the generator to produce coherent questions around them. 

But if there's a more "network intelligent" way to grab those triples we can build into your algorithm, I'm all for it!

Reply:
@gallupBenRyan let's discuss at 4p ET quickly, so we can move this forward for Mitch.